docker/sentinel-compose.yml

```
@@ -0,0 +1,98 @@
+      - 26380:26380
+    entrypoint: ["/bin/bash", "-c"]
+    command: |
+      touch /data/sentinel.conf
+      echo "sentinel monitor master redis-server-1 6380 2" >> /data/sentinel.conf
+      echo "sentinel announce-hostnames yes"               >> /data/sentinel.conf
+      echo "sentinel announce-ip sentinel-2"               >> /data/sentinel.conf
+      echo "sentinel resolve-hostnames yes"                >> /data/sentinel.conf
+      echo "sentinel announce-port 26380"                  >> /data/sentinel.conf
+      echo "port 26380"                                    >> /data/sentinel.conf
+      redis-server /data/sentinel.conf --sentinel
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)